examples/visualize/tree/container_tree/README.md

```diff
@@ -6,7 +6,7 @@ The functions under analysis will help you show the contents of an image (folder
 
 This will open up something that looks like this:
 
-![../../img/files.png](../../img/files.png)
+![container tree](/img/files.png)
```

Comment:
If the fix is intended for just GitHub (and you can confirm this will work to represent the root of the repository) this should be good to render in the README - I had a relative path before in the case that it was on a GitHub pages server (and it was a few levels too shallow).

Reply:
Good catch, I didn't think to check that. Looking now

Reply:
Let me know when you've had a chance to look!

Reply:
I apologize for the lapse in communication. Checking it now.